### PR TITLE
Plugin Work Queue will now write GraphDescriptions to topic "generated-graphs"

### PR DESCRIPTION
**Pull request by wimax-grapl**
### Which issue does this PR correspond to?
.never made an issue for this

### What changes does this PR make to Grapl? Why?
We discovered https://github.com/grapl-security/grapl/pull/1811 would not work because of too many producers. 
Now, when a plugin sidecar acknowledges its work to PWQ, PWQ will write the generated graph to a kafka topic

### How were these changes tested?
not quite there yet. SOON! We need the generator-dispatcher in order for work to show up in a generator.

### CODE REVIEW

File: nomad/grapl-core.nomad [hunk 1]
```diff
@@ -1233,6 +1233,13 @@ job "grapl-core" {
         # Hardcoded, but makes little sense to pipe up through Pulumi
         PLUGIN_WORK_QUEUE_HEALTHCHECK_POLLING_INTERVAL_MS = 5000
 
+        KAFKA_BOOTSTRAP_SERVERS = var.kafka_bootstrap_servers
+        KAFKA_SASL_USERNAME     = var.kafka_credentials["plugin-work-queue"].sasl_username
+        KAFKA_SASL_PASSWORD     = var.kafka_credentials["plugin-work-queue"].sasl_password
+
+        KAFKA_PRODUCER_TOPIC_GENERATOR = "generated-graphs"
+        # KAFKA_PRODUCER_TOPIC_ANALYZER = "TODO"
```
Comment (wimax-grapl) [inline nomad/grapl-core.nomad:1241]: not in this PR, more like a "hey don't forget to do this eventually"

Comment (jgrillo-grapl) [reply nomad/grapl-core.nomad:1241]: nit: could these be named `GENERATOR_KAFKA_PRODUCER_TOPIC` and `ANALYZER_KAFKA_PRODUCER_TOPIC`? When I see `KAFKA_PRODUCER_TOPIC_GENERATOR` I think it's something responsible for generating kafka producer topics.

File: src/rust/kafka/src/config.rs [hunk 2]
```diff
@@ -12,6 +14,18 @@ pub struct ConsumerConfig {
     pub topic: String,
 }
 
+impl ConsumerConfig {
+    // In some test cases, where we need to consume from >1 topic,
```
Comment (jgrillo-grapl) [inline src/rust/kafka/src/config.rs:18]: these should be docs comments `///`

File: src/rust/kafka/src/config.rs [hunk 3]
```diff
@@ -23,3 +37,20 @@ pub struct ProducerConfig {
     #[clap(long, env = "KAFKA_PRODUCER_TOPIC")]
     pub topic: String,
 }
+
+impl ProducerConfig {
+    // In some applications, where we need to produce to >1 topic,
```
Comment (jgrillo-grapl) [inline src/rust/kafka/src/config.rs:42]: `///`

**Review by jgrillo-grapl — APPROVED** [review]

**Pull request merged**